js/router/SharedNavigator.js

```diff
@@ -78,6 +79,7 @@ const SharedNavigator = React.createClass({
       title: '',
       isLoading: true,
       isAuthenticated: false,
+      newLogin: false,
```

Comment:
This logic prevents the user from reloading every single form, trigger, and question available on Parse every time they load the app. But it will allow them to gather that data the first time log in without having to re-accept/decline.